### ISSUE TITLE
Make it easy to get to organizers slides

### ISSUE THREAD

**Issue by adamchainz** [djangolondon/djangolondon.github.io#61]
We could do with a page that lists them all so they're easy to access when setting up the meetup or browsing thes ite.

**Comment by marcoala** [comment]
Shall we add another collection for them in `_config.yml`?

**Comment by adamchainz** [comment]
Yeah, I think there's some other stuff to do for that as well though
